src/components/Buttons/FutureButton/FutureButton.tsx

```diff
@@ -39,16 +42,16 @@ const FutureButton: React.FC<FutureButtonProps> = ({
 
 	return (
 		<button
-			className={`${styles.futureButton} ${isSelected ? styles.selected : ''} ${
-				glow ? styles.glow : ''
-			} ${loading ? styles.Loading : ''} ${alt ? styles.Alt : ''} ${
-				className ? className : ''
-			}`}
+			className={`${styles.futureButton} ${
+				glow ? ' ' : styles.futureButtonUnActive
```

Comment:
A couple minor things: 
1. I would use `''` instead of `' '` here because if you're adding a bunch of spaces to the class list it'll get quite unruly
2. "UnActive" is slightly incorrect grammatically, it should be "Inactive"

The above aren't showstoppers but would be good to fix